data/src/main/java/org/oppia/android/data/backends/gae/NetworkSettings.kt

```diff
@@ -7,7 +7,7 @@ object NetworkSettings {
 
   /** DEVELOPER URL which connects to development server */
   // TODO(#3623): Move this to DI graph
-  private const val DEVELOPER_URL = "https://oppia.org"
+  private const val DEVELOPER_URL = "http://localhost:8181/"
```

Comment:
This isn't what I meant. We still need to point to prod Oppia by default; only in the test app should we be switching this. To do that, we need to change this to be injected via a Dagger module in the same way as the other constants you're swapping out below for the test app.

Reply:
Done